pptx/dml/line.py

```diff
@@ -100,3 +100,68 @@ def _get_or_add_ln(self):
     @property
     def _ln(self):
         return self._parent.ln
+
+
+
+class LineStyle(object):
```

Comment:
Also, a lot of this code look identical to the LineFormat code. What about making this a child of LineFormat and overriding the `__set__` function, not sure if that would work.

Reply:
So this wasn't done in order to be Read Only, it was done because `LineFormat` requires a parent object (like a shape) and a single child `ln` object.  All of the `LineFormat` objects rely on the self._get_or_add_ln() which is references self._parent.  If I inherited from `LineFormat` then I would have to overwrite basically every method and also remove a few and that just didn't make sense.  That was my first approach though